qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -5,6 +5,7 @@
     gagent_name = "org.qemu.guest_agent.0"
     gagent_install_cmd = "yum install -y qemu-guest-agent"
     gagent_start_cmd = "service qemu-guest-agent start"
+    gagent_restart_cmd = "service qemu-guest-agent restart"
```

Comment:
Guess we need to use systemctl for RHEL8 support.

Reply:
Seems rhel6 doesn't support systemctl,do we support rhel6 later?

Reply:
Yes, so it could be:
```
gagent_restart_cmd = systemctl ...
RHEL.6:
   gagent_restart_cmd = service ...
```

Reply:
Update and the line27 cmd is for windows guest(the comment below).
Thank you